test/init/init.test.js

```diff
@@ -0,0 +1,25 @@
+const init = require('../../dist/src/cmds/init');
+const initializeModule = require('../../dist/src/cmds/init/initHandler');
+const buildModule = require('../../dist/src/cmds/build');
+
+describe('init', () => {
+  describe('init', () => {
+
+    afterEach(() => {
+      jest.clearAllMocks();
+      jest.restoreAllMocks();
+    });
+
+    it('console logs if successful', async () => {
+      const mockArgv = { foo: 'bar' };
```

Comment:
Since this is only a single test case, we don't need to define a getter, but we should destructure `mockArgv` to instantiate a new object in the call to `init.handler()`.